frontend/lib/sanityClient.ts

```diff
@@ -5,4 +5,5 @@ export const sanityClient = createClient({
   dataset: 'test', 
   useCdn: true,
   apiVersion: '2023-09-01',
+  token: process.env.NEXT_PUBLIC_SANITY_API_TOKEN || undefined,
```

Comment:
Please let Boris know about this

Reply:
it can simply be called SANITY_API_TOKEN

Reply:
@captainboris Yeah, and we need CICD for sanity as well but I suspect it would be fairly simple to do